Build hadoop-run failed. "COPY failed: stat /var/lib/docker/tmp/docker-builder814011495/hadoop/hadoop-2.7.2.tar.gz: no such file or directory"

My dev-box:
```
        "HostConfig": {
            "Binds": [
                "/mnt/pathConfiguration:/cluster-configuration",
                "/var/lib/docker:/var/lib/docker",
                "/var/run/docker.sock:/var/run/docker.sock",
                "/pathHadoop:/pathHadoop"
            ],
```

My `services-configuration.yaml`:
```
hadoop:
  custom-hadoop-binary-path: /pathHadoop
  hadoop-version: 2.7.2

```

My cmd in dev-box:
`./paictl.py image build -p /cluster-configuration -n hadoop-run`

Related build log:
```
Step 3/21 : RUN apt-get -y install zookeeper libsnappy-dev
 ---> Using cache
 ---> f8217e9d7ff6
Step 4/21 : RUN rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> c9e2e8ab132c
Step 5/21 : COPY hadoop/hadoop-2.7.2.tar.gz /usr/local/
COPY failed: stat /var/lib/docker/tmp/docker-builder814011495/hadoop/hadoop-2.7.2.tar.gz: no such file or directory
2018-06-27 07:22:39,194 [ERROR] - paiLibrary.common.linux_shell : An error occurs, when building your image [ hadoop-run ]
```

--- a/cluster-configuration/services-configuration.yaml
+++ b/cluster-configuration/services-configuration.yaml
@@ -53,7 +53,8 @@ cluster:
 hadoop:
   # custom_hadoop_binary_path is required to build hadoop-ai.
   # More about hadoop-ai please follow the link: https://github.com/Microsoft/pai/tree/master/hadoop-ai.
-  custom-hadoop-binary-path: /pathHadoop/
+  # Notice: the name should be hadoop-{hadoop-version}.tar.gz
+  custom-hadoop-binary-path: /pathHadoop/hadoop-2.7.2.tar.gz
   hadoop-version: 2.7.2
   # Step 1 of 4 to set up Hadoop queues.
   # Define all virtual clusters, equivalent concept of Hadoop queues:

--- a/pai-management/doc/how-to-write-pai-configuration.md
+++ b/pai-management/doc/how-to-write-pai-configuration.md
@@ -206,7 +206,8 @@ Users can browse to https://hub.docker.com/r/openpai to see all the repositories
 ```YAML
 hadoop:
   # custom_hadoop_binary_path specifies the path PAI stores the custom built hadoop-ai
-  custom-hadoop-binary-path: /pathHadoop/
+  # Notice: the name should be hadoop-{hadoop-version}.tar.gz
+  custom-hadoop-binary-path: /pathHadoop/hadoop-2.7.2.tar.gz
   hadoop-version: 2.7.2
   virtualClusters:
     default:

--- a/pai-management/quick-start/services-configuration.yaml.template
+++ b/pai-management/quick-start/services-configuration.yaml.template
@@ -53,7 +53,8 @@ cluster:
 hadoop:
   # When needed, hadoop-ai would be downloaded and built under 'custom-hadoop-binary-path'.
   # More about hadoop-ai please follow the link: https://github.com/Microsoft/pai/tree/master/hadoop-ai.
-  custom-hadoop-binary-path: /pathHadoop/
+  # Notice: the name should be hadoop-{hadoop-version}.tar.gz
+  custom-hadoop-binary-path: /pathHadoop/hadoop-2.7.2.tar.gz
   hadoop-version: 2.7.2
   # Step 1 of 4 to set up Hadoop queues.
   # Define all virtual clusters, equivalent concept of Hadoop queues:

